pkg/apps/registry/deployconfig/etcd/etcd.go

```diff
@@ -139,6 +139,6 @@ func (r *StatusREST) Get(ctx apirequest.Context, name string, options *metav1.Ge
 }
 
 // Update alters the status subset of an deploymentConfig.
-func (r *StatusREST) Update(ctx apirequest.Context, name string, objInfo rest.UpdatedObjectInfo) (runtime.Object, bool, error) {
-	return r.store.Update(ctx, name, objInfo)
+func (r *StatusREST) Update(ctx apirequest.Context, name string, objInfo rest.UpdatedObjectInfo, createValidation rest.ValidateObjectFunc, updateValidation rest.ValidateObjectUpdateFunc) (runtime.Object, bool, error) {
```

Comment:
we could have had https://github.com/kubernetes/kubernetes/pull/55240

Reply:
> we could have had kubernetes/kubernetes#55240

I think that just shuffles it and may even make it possible for me to accidentally miss one.  This sucks, but it beats missing one.

Reply:
I agree. Bundling the validation into the object info still seems weird to me.